### ISSUE TITLE
Grimy Utility - Quick Cast

### ISSUE THREAD

**Issue by TytchUK** [zhongjiedong/Tsukiro#141]
### What happened?

Casting courage via hotkeys (Grimy) causes major relationship loss.

If you cast it directly (equip spell) no loss of relationship. This is potentially the reason followers have been fighting amongst selves after NPC fights. Potentially causing issues with quests as Katana has still not offered her feather or offered 1st quest to go to Whiterun.

On the back of the 'bug' with fast casting bound weapons in previous report (no dispel when sheathed) it seems Grimy is causing a few issues, not the WJ list

https://youtu.be/F1nGUzj4iN4

### Modlist version

1.05

**Comment by zhongjiedong** [comment]
can you check if it happens with rally too?

**Comment by TytchUK** [comment]
Yes it is, but it is also happening with normal cast as well, but in-consistently. I will test more when I get home on the 2nd Sept (no doubt there will be an update by then anyway) I think this may have been an old save from 1.03 or 1.04 where I saved in the opening room before entering Skyrim. Will see what's up when I am back and start a fresh game.

Thanks
